cicd/2-cicd/cicd.template.yml

```diff
@@ -394,6 +394,36 @@ Resources:
                       - '" }'
                   Capabilities: CAPABILITY_AUTO_EXPAND
                   RoleArn: !Sub arn:aws:iam::${AWS::AccountId}:role/admin/CloudFormationService
+              - Name: app-demo-deploy
+                ActionTypeId:
+                  Category: Deploy
+                  Owner: AWS
+                  Version: 1
+                  Provider: CloudFormation
+                InputArtifacts:
+                  - Name: appBuildResults
+                # The value of `Configuration` must be an object with String (or simple type) properties
+                Configuration:
+                  StackName:
+                    !If [
+                      TargetsMainBranch,
```

Comment:
Probably a question for @cat5inthecradle - is it possible/should it be possible to target a branch other than `main` for a production deploy?